### PR TITLE
teams: integrate hidden ratchets

### PR DESCRIPTION
**Pull request by maxtaco**
- for FTL and slow loader, pull down ratchets, with blinding keys
- persist those ratchets to the local store
- some small happy-path testing, but took a lot of fiddling to get everything to work (since the server does a lot of error checking)

### CODE REVIEW

**Comment by maxtaco** [comment]
Bump @heronhaye in case it got lost in the mix. Though no rush.

**Comment by maxtaco** [comment]
@heronhaye ping

File: go/teams/hidden/ratchet.go [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+}
+
+func (r SCTeamRatchet) String() string {
+	return hex.EncodeToString(r[:])
+}
+
+// UnmarshalJSON is implicitly used in chain_parse.go move SCTeamRatchets into and out of JSON
+// from the hidden team chain.
+func (r *SCTeamRatchet) UnmarshalJSON(b []byte) error {
+	unquoted := keybase1.UnquoteBytes(b)
```
Comment (heronhaye) [inline go/teams/hidden/ratchet.go]: Confused about this quoting and unquoting. Are we expecting double encoded JSON here?

**Comment by maxtaco** [comment]
merging over CI flake

**Pull request merged**